### DISCUSSION TITLE
[Suggestion] Isolate "Reverse Mute" Audio Streams

### DISCUSSION THREAD

**Discussion by SpyderZT** [File-New-Project/EarTrumpet#657] [Ideas]
I'm not sure if that's the best way to word that, but it would be awesome if you were able to "Reverse Mute" / Isolate (Perhaps by Right Clicking on a Source Icon instead of Left Clicking) audio streams. At which point, only similarly right clicked streams would be permitted to output sound. For Example, if I'm listening to my media player and I've right clicked that stream and the system sounds stream, even if I opened a browser tab playing music, no sound would play unless I likewise added that stream. So a global mute with selected exceptions.

**Comment by riverar** [comment]
So if I understand correctly, you want the ability to "mute all except selected" right?

**Comment by SpyderZT** [comment]
That would certainly be a better way to say it. :P Specifically the difference between just muting everything and only selecting certain streams would be the fact that any new streams, whether or not the application has seen them before (Since I've noticed it saves audio volume for streams it's seen) would start out muted while set up this way.
